test/test_s3_utils.py

```diff
@@ -92,28 +98,31 @@ def test_regression_s3_utils_short_name_c4_706():
         s3Utils(env="mastertest")
 
 
-def _env_is_up_and_healthy(env):
-    env_url = beanstalk_utils.get_beanstalk_real_url(env)
-    health_page_url = f"{env_url}/health?format=json"
-    return requests.get(health_page_url).status_code == 200
+def _env_is_probably_up_and_healthy(env):
+    ignored(env)
+    return True  # not quite right, but
+    # This code won't work in container systems...
+    # env_url = beanstalk_utils.get_beanstalk_real_url(env)
+    # health_page_url = f"{env_url.rstrip('/')}/health?format=json"
+    # return requests.get(health_page_url).status_code == 200
```

Comment:
Not clear on why you can't do this in the new paradigm. GLOBAL_ENV_BUCKET should be set, and thus we should be able to resolve the portal URL. The system does not function without GLOBAL_ENV_BUCKET, and we should not write code that tries to handle things when it is absent. Rather we should detect it is absent and report an error.

Reply:
I've revised the way this works, but it's doing the proper test and doing the right thing. In fact, it was going to the wrong environment. It's better now.